exo-go/src/cmd/configure.go

```diff
@@ -110,8 +107,117 @@ var configureCreateCmd = &cobra.Command{
 	},
 }
 
+var configureUpdateCmd = &cobra.Command{
+	Use:   "update",
+	Short: "Updates secret key entries in remote secrets store",
+	Long:  "Updates secret key entries in remote secret store. Keys should already exist.",
+	Run: func(cmd *cobra.Command, args []string) {
+		if printHelpIfNecessary(cmd, args) {
+			return
+		}
+		fmt.Print("We are about update keys in the remote store!\n\n")
+
+		secretsBucket, awsRegion, err := getSecretsConfig()
+		if err != nil {
+			log.Fatalf("Cannot update secrets: %s", err)
+		}
+
+		secretsString, err := aws.ReadSecrets(secretsBucket, awsRegion)
+		if err != nil {
+			log.Fatalf("Cannot read secrets: %s", err)
+		}
+		existingSecrets := types.NewSecrets(secretsString)
+		existingSecretKeys := existingSecrets.Keys()
+		fmt.Print("Existing secrets:\n\n")
+		prettyPrint(existingSecrets)
+
+		newSecrets := map[string]string{}
+		ok := true
+		for ok {
+			i := prompt.Choose("Select secret keys to update", existingSecretKeys)
+			value := prompt.StringRequired(fmt.Sprintf("Secret value for %s", existingSecretKeys[i]))
+			newSecrets[existingSecretKeys[i]] = value
+			existingSecretKeys = append(existingSecretKeys[:i], existingSecretKeys[i+1:]...)
+			ok = prompt.Confirm("Do you have more keys to update?")
+		}
+
+		if len(newSecrets) > 0 {
+			fmt.Print("\nYou are updating these secrets:\n\n")
+			prettyPrint(newSecrets)
+
+			if ok := prompt.Confirm("Do you want to continue?"); ok {
+				err = aws.MergeAndWriteSecrets(existingSecrets, newSecrets, secretsBucket, awsRegion)
+				if err != nil {
+					log.Fatalf("Cannot update secrets: %s", err)
+				}
+			} else {
+				fmt.Println("Secret update abandoned.")
+			}
+		}
+	},
+}
+
+var configureDeleteCmd = &cobra.Command{
+	Use:   "delete",
+	Short: "Deletes secrets from the remote secrets store",
+	Long:  "Deletes secrets from the remote secrets store. Ignores any keys passed in that don't exist on the remote store.",
+	Run: func(cmd *cobra.Command, args []string) {
+		if printHelpIfNecessary(cmd, args) {
+			return
+		}
+		fmt.Print("We are about to delete secrets from the secret store...\n\n")
+
+		secretsBucket, awsRegion, err := getSecretsConfig()
+		if err != nil {
+			log.Fatalf("Cannot delete secrets: %s", err)
+		}
+
+		secretsString, err := aws.ReadSecrets(secretsBucket, awsRegion)
+		if err != nil {
+			log.Fatalf("Cannot read secrets: %s", err)
+		}
+		existingSecrets := types.NewSecrets(secretsString)
+		existingSecretKeys := existingSecrets.Keys()
+		fmt.Print("Existing secrets:\n\n")
+		prettyPrint(existingSecrets)
```

Comment:
This chunk of code (definition of ReadSecrets to prettyPrint) appears duplicated between update and delete. Getting the bucket could even be included in that as the error message should probably say `Cannot get AWS secrets config:` instead of `Cannot update/delete secrets:`